tests/stubtest_allowlists/py3_common.txt

```diff
@@ -355,6 +355,7 @@ _?ctypes.Array._length_ # _length_ is abstract, https://github.com/python/typesh
 _pydecimal.*  # See comments in file
 argparse.ArgumentParser.__init__  # stubtest doesn't recognise the runtime default (a class) as being compatible with a callback protocol (the stub annotation)
 argparse.Namespace.__getattr__  # The whole point of this class is its attributes are dynamic
+argparse._VersionAction.__init__  # "help" argument default changed in 3.11+ point releases, remove entry when dropping 3.10 support
```

Comment:
We could instead copy this to the individual version allowlists, but considering this was changed in a point release (and not having it made yet into the Python 3.12 point release used at GitHub), I think it's safer to have it here – at least for now.